CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 * Kotlin 2.0.21
 * [Gradle Plugin] Gradle 8.10.2
 * [Gradle Plugin] Android Gradle Plugin 8.4.2
+* Support for editable text in accessibility snapshots
```

Comment:
move to Unreleased.  this section is for an already-released version.

Reply:
oops!